doc/manual/macros

```diff
@@ -66,6 +66,7 @@ to perform useful operations. The current list is
 	%trace		toggle print of debugging information before/after
 			expansion
 	%dump		print the active (i.e. non-covered) macro table
+	%getncpus	return the number of CPUs
```

Comment:
It's not a function, it doesn't really "return" anything, it expands to the number of CPUs, and more precisely CPUs *available to the current process*, which is not necessarily the same thing as total number of processors on the system. Fix those and its an obvious keeper.